diff -ru --strip-trailing-cr iup.orig/srcole/tLegacy.h iup/srcole/tLegacy.h
--- iup.orig/srcole/tLegacy.h	2013-02-02 13:03:12.350817000 +0100
+++ iup/srcole/tLegacy.h	2013-02-02 17:59:50.866833900 +0100
@@ -91,8 +91,7 @@
 typedef POINTL * PPOINTL;
 #endif  //PPOINTL
 
-
-#ifndef _WIN32
+#if !defined(_WIN32) && !defined(__CYGWIN__)
 #ifndef OLECHAR
 typedef char OLECHAR;
 typedef OLECHAR FAR* LPOLESTR;
